dpnp/dpnp_iface_linearalgebra.py

```diff
@@ -194,21 +194,30 @@ def inner(x1, x2, **kwargs):
     return call_origin(numpy.inner, x1, x2, **kwargs)
 
 
-def kron(x1, x2, **kwargs):
+def kron(a, b):
```

Comment:
please don't use single letter variables names

Reply:
previously we decided to use the same names (as in original numpy) in python interface to be compliant with original numpy and its documentation, that is why I made these changes